.github/workflows/release-please.yml

```diff
@@ -0,0 +1,13 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
```

Comment:
would it be ok to call it 'change log' instead of 'release-please' so that we can tell what it is quickly in the actions list?

Reply:
Although maybe it's self explanatory enough? Thoughts? 
![image](https://user-images.githubusercontent.com/28784886/217625387-0aa99ffc-b36b-4b59-bd29-608881398ce3.png)

Reply:
Happy to change it to change-log I think release-please might be more encompassing though because the action covers publishing Release versions on GitHub which also include change-logs.

Reply:
I am fine with that too.